content/02.introduction.md

```diff
@@ -9,7 +9,8 @@ At the time of the first situation report [@url:https://www.who.int/docs/default
 One week later, 4593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf].
 On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf].
 By WHO Situation Report 61, released on March 20, 2020, 266,073 confirmed cases had been reported worldwide, with cases on every continent except Antarctica [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200321-sitrep-61-covid-19.pdf].
```

Comment:
```suggestion
```
If you didn't guess, I wrote this during the initial configuring on March 20th -- that's the only reason March 20th was significant :) Thanks for updating it!